app/(chat)/api/chat/route.ts

```
@@ -49,44 +46,44 @@ export async function POST(request: Request) {
     if (typeof firstPart === 'string') {
       messageText = firstPart;
     } else if (firstPart && 'type' in firstPart && firstPart.type === 'text' && 'text' in firstPart) {
       messageText = firstPart.text;
     }
-    
+
     // Check for API key in the message - more permissive pattern to catch URLs, etc.
     const apiKeyMatch = messageText.match(/^API[_\s]?KEY[:\s]+(.+)$/i);
-    
+
     if (apiKeyMatch && apiKeyMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)